en/style-guide.adoc

```diff
@@ -10,15 +10,15 @@ This document describes the _official coding style used in the Red source code_,
 
 As Red/System is a dialect of Red, it shares the same syntax and coding style rules. Specific Red/System rules are marked as such.
 
-The objectives of the following rules are to maximise readability, including keeping an optimum number of lines of code visible on-screen while minimizing the need for comments.
+The objective of the following rules is to maximise readability, including keeping an optimum number of lines of code visible on-screen while minimizing the need for comments.
```

Comment:
"The purpose of the following rules is to ..., by keeping ..." is IMO more appropriate here. "Objective" sounds too active, as if rules maximize readability themselves, without any need for a programmer to follow them.

Also, I think "optimum" should rather be "optimal".